lib/internal/bootstrap/loaders.js

```diff
@@ -155,6 +155,7 @@ function NativeModule(id) {
   this.loaded = false;
   this.loading = false;
   this.canBeRequiredByUsers = !id.startsWith('internal/');
+  this.defaultOnly = false;
```

Comment:
`defaultOnly` here is set for internal modules to ensure that they do not get synthetic exports. For some modules like policy the synthetic exports iteration throws errors.

Reply:
Sure, will do.